services/121-service/test/onafriq/subscribe-webhook.test.ts

```diff
@@ -0,0 +1,43 @@
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import {
+  getAccessToken,
+  getServer,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+
+describe('Subscribe webhook', () => {
+  const programId = 1;
+  let accessToken: string;
+
+  it('should successfully subscribe webhook', async () => {
+    // Arrange
+    await resetDB(SeedScript.onafriqProgram, __filename);
+    accessToken = await getAccessToken();
+
+    // Act
+    const response = await getServer()
+      .post(`/fsps/onafriq/webhook/subscribe/programs/${programId}`)
+      .set('Cookie', [accessToken])
+      .send();
+
```

Comment:
This can be a function in one of the helpers or in this file

Reply:
sure agree given it's used twice. I moved it to helper (and reorganized some helper files slightly)